### PR TITLE
Validation issues | 1 (BULK)

### PR DESCRIPTION
**Pull request by damabe**
This is a bulk update.

This PR is to fix issues related to build validation messages.

If these changes are not appropriate, please let me know. I'm only trying to clean up validation issues in this repo. So I can easily close this PR or modify it in whatever way is best for the content.

### CODE REVIEW

**Comment by damabe** [comment]
Global effort to fix build validation errors

@Rick-Anderson @mcleblanc 

The Content & Learning team is fixing build validation errors on docs.microsoft.com for the rest of H2. This will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. Please review within five business days and merge or comment in the PR with any changes you’d like to see. Thank you!

**Comment by jmart1428** [comment]
@Rick-Anderson @mcleblanc Please let us know if you'd like to see any additional changes in this PR. If no changes are required, you are welcome to merge. Thank you for your feedback!

**Review by wadepickett — CHANGES_REQUESTED** [review]
Thanks @damabe!  We need here is to know who to assign these soshir/shirhatti (Sourabh Shirhatti) topics to in bulk as ms.author, since Sourabh Shirhatti moved on.

I don't think assigning them to @mairaw is correct.  She is not on the IIS team.  She has touched them in the past as a content developer when she was once in that role.

(I'm not assigned to these either, I could see however the PR's sitting here so I thought I would jump in to try to help resolve if I could.)  

On your PR #924 I asked John Hart if he know who these IIS topics belonging to soshir/shirattie (Sourabh Shirhatti) should be assigned to now.  Hopefully John might know who.

**Comment by mairaw** [comment]
For sure, I shouldn't be the author for any IIS docs. I also think @John-Hart is probably the best contact here.

**Comment by wadepickett** [comment]
@John-Hart, is there someone that all Sourabh Shirhatti topics can be assigned to for "ms.author", or at least the ones that are active in some way?

**Comment by John-Hart** [comment]
@wadepickett you can use John-Hart as the ms.author

File: iis/extensions/database-manager-reference/queryresult-columnmetadata-property-microsoft-web-management-databasemanager.md [hunk 1]
```diff
@@ -2,12 +2,12 @@
 title: QueryResult.ColumnMetadata Property (Microsoft.Web.Management.DatabaseManager)
 description: "This article outlines syntax, remarks, examples, and permissions for the QueryResult.ColumnMetadata Property, which returns a list of column metadata for a query result."
 TOCTitle: ColumnMetadata Property
-description: This article contains syntax, examples, and permission information for the QueryResult.ColumnMetadata property.
 ms:assetid: P:Microsoft.Web.Management.DatabaseManager.QueryResult.ColumnMetadata
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.management.databasemanager.queryresult.columnmetadata(v=VS.90)
 ms:contentKeyID: 20476453
 ms.date: 05/02/2012
 mtps_version: v=VS.90
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/database-manager-reference/queryresult-columnmetadata-property-microsoft-web-management-databasemanager.md:10]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-currentsegmentindexproperty-field-microsoft-web-media-smoothstreaming.md [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+title: SmoothStreamingMediaElement.CurrentSegmentIndexProperty Field - Microsoft.Web.Media.SmoothStreaming
 description: Describes the CurrentSegmentIndexProperty field and provides the field's syntax, version information, and permissions.
 TOCTitle: CurrentSegmentIndexProperty Field
-description: Represents a dependency property.
 ms:assetid: F:Microsoft.Web.Media.SmoothStreaming.SmoothStreamingMediaElement.CurrentSegmentIndexProperty
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreaming.smoothstreamingmediaelement.currentsegmentindexproperty(v=VS.90)
 ms:contentKeyID: 31469166
 ms.date: 05/02/2012
 mtps_version: v=VS.90
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-currentsegmentindexproperty-field-microsoft-web-media-smoothstreaming.md:10]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-currentsegmentindexproperty-field-microsoft-web-media-smoothstreaming_1.md [hunk 3]
```diff
@@ -7,6 +7,7 @@ ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreami
 ms:contentKeyID: 46307547
 ms.date: 05/31/2012
 mtps_version: v=VS.95
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-currentsegmentindexproperty-field-microsoft-web-media-smoothstreaming_1.md:10]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-naturalvideoheightproperty-field-microsoft-web-media-smoothstreaming.md [hunk 4]
```diff
@@ -1,12 +1,12 @@
...
-title: SmoothStreamingMediaElement.NaturalVideoHeightProperty Field (Microsoft.Web.Media.SmoothStreaming)
-description: The NaturalVideoHeightProperty Field represents a dependency property that specifies the NaturalHeight()()()() property. This article details syntax, version information, and permissions.
+title: SmoothStreamingMediaElement.NaturalVideoHeightProperty Field - Microsoft.Web.Media.SmoothStreaming
 TOCTitle: NaturalVideoHeightProperty Field
 description: "The NaturalVideoHeightProperty field represents a dependency property that specifies the NaturalHeight property."
 ms:assetid: F:Microsoft.Web.Media.SmoothStreaming.SmoothStreamingMediaElement.NaturalVideoHeightProperty
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreaming.smoothstreamingmediaelement.naturalvideoheightproperty(v=VS.90)
 ms:contentKeyID: 23960945
 ms.date: 05/02/2012
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-naturalvideoheightproperty-field-microsoft-web-media-smoothstreaming.md:9]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-naturalvideowidthproperty-field-microsoft-web-media-smoothstreaming.md [hunk 5]
```diff
@@ -1,11 +1,12 @@
 ---
-title: SmoothStreamingMediaElement.NaturalVideoWidthProperty Field (Microsoft.Web.Media.SmoothStreaming)
+title: SmoothStreamingMediaElement.NaturalVideoWidthProperty Field - Microsoft.Web.Media.SmoothStreaming
 description: This article shows syntax and version information of the NaturalVideoWidthProperty Field, which represents a dependency property that specifies the NaturalVideoWidth property.
 TOCTitle: NaturalVideoWidthProperty Field
 ms:assetid: F:Microsoft.Web.Media.SmoothStreaming.SmoothStreamingMediaElement.NaturalVideoWidthProperty
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreaming.smoothstreamingmediaelement.naturalvideowidthproperty(v=VS.90)
 ms:contentKeyID: 23961184
 ms.date: 05/02/2012
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/smoothstreamingmediaelement-naturalvideowidthproperty-field-microsoft-web-media-smoothstreaming.md:9]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/streamupdatedeventargs-class-microsoft-web-media-smoothstreaming.md [hunk 6]
```diff
@@ -1,12 +1,12 @@
 ---
 title: StreamUpdatedEventArgs Class (Microsoft.Web.Media.SmoothStreaming)
-description: Arguments for the StreamSelected, ChunkAdded and TracksSelected events.
 TOCTitle: StreamUpdatedEventArgs Class
 description: "The StreamUpdatedEventArgs class arguments for the StreamSelected, ChunkAdded and TracksSelected events."
 ms:assetid: T:Microsoft.Web.Media.SmoothStreaming.StreamUpdatedEventArgs
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreaming.streamupdatedeventargs(v=VS.90)
 ms:contentKeyID: 31469273
 ms.date: 05/02/2012
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/streamupdatedeventargs-class-microsoft-web-media-smoothstreaming.md:9]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/streamupdatedeventargs-class-microsoft-web-media-smoothstreaming_1.md [hunk 7]
```diff
@@ -1,12 +1,13 @@
...
-title: StreamUpdatedEventArgs Class (Microsoft.Web.Media.SmoothStreaming)
+title: StreamUpdatedEventArgs Class - Microsoft.Web.Media.SmoothStreaming
 description: Describes the StreamUpdatedEventArgs class and provides the class' inheritance hierarchy, namespace, assembly, syntax, and version information.
 TOCTitle: StreamUpdatedEventArgs Class
 ms:assetid: T:Microsoft.Web.Media.SmoothStreaming.StreamUpdatedEventArgs
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreaming.streamupdatedeventargs(v=VS.95)
 ms:contentKeyID: 46307967
 ms.date: 05/31/2012
 mtps_version: v=VS.95
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/streamupdatedeventargs-class-microsoft-web-media-smoothstreaming_1.md:10]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/streamupdatedlisteventargs-class-microsoft-web-media-smoothstreaming.md [hunk 8]
```diff
@@ -1,12 +1,13 @@
...
-title: StreamUpdatedListEventArgs Class (Microsoft.Web.Media.SmoothStreaming)
+title: StreamUpdatedListEventArgs Class - Microsoft.Web.Media.SmoothStreaming
 description: StreamUpdatedListEventArgs Class is a list of Stream Updated Event Args.
 TOCTitle: StreamUpdatedListEventArgs Class
 ms:assetid: T:Microsoft.Web.Media.SmoothStreaming.StreamUpdatedListEventArgs
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreaming.streamupdatedlisteventargs(v=VS.90)
 ms:contentKeyID: 31469191
 ms.date: 05/02/2012
 mtps_version: v=VS.90
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/streamupdatedlisteventargs-class-microsoft-web-media-smoothstreaming.md:10]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/smooth-streaming-client/streamupdatedlisteventargs-class-microsoft-web-media-smoothstreaming_1.md [hunk 9]
```diff
@@ -7,6 +7,7 @@ ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.media.smoothstreami
 ms:contentKeyID: 46307956
 ms.date: 05/31/2012
 mtps_version: v=VS.95
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/smooth-streaming-client/streamupdatedlisteventargs-class-microsoft-web-media-smoothstreaming_1.md:10]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/database-manager-reference/dependenciesmissingexception-constructor-serializationinfo-streamingcontext-microsoft-web-management-databasemanager.md [hunk 10]
```diff
@@ -1,11 +1,12 @@
 ---
 title: DependenciesMissingException Constructor (SerializationInfo, StreamingContext) (Microsoft.Web.Management.DatabaseManager)
-description: This article contains information about syntax and permissions for the DependenciesMissingException constructor.
+description: In this article, you can learn about syntax and permissions for creating a new instance of the DependenciesMissingException constructor.
 TOCTitle: DependenciesMissingException Constructor (SerializationInfo, StreamingContext)
 ms:assetid: M:Microsoft.Web.Management.DatabaseManager.DependenciesMissingException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.management.databasemanager.dependenciesmissingexception.dependenciesmissingexception(v=VS.90)
 ms:contentKeyID: 22049490
 ms.date: 05/02/2012
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/database-manager-reference/dependenciesmissingexception-constructor-serializationinfo-streamingcontext-microsoft-web-management-databasemanager.md:9]:
```suggestion
ms.author: johnhart
```

File: iis/extensions/database-manager-reference/dependenciesmissingexception-constructor-microsoft-web-management-databasemanager_1.md [hunk 11]
```diff
@@ -1,11 +1,12 @@
...
-title: DependenciesMissingException Constructor  (Microsoft.Web.Management.DatabaseManager)
-description: This article contains information about syntax and permissions for the DependenciesMissingException constructor.
+title: DependenciesMissingException Constructor - Microsoft.Web.Management.DatabaseManager
+description: This article contains information about syntax and permissions for the DependenciesMissingException constructor. 
 TOCTitle: DependenciesMissingException Constructor
 ms:assetid: M:Microsoft.Web.Management.DatabaseManager.DependenciesMissingException.#ctor
 ms:mtpsurl: https://msdn.microsoft.com/library/microsoft.web.management.databasemanager.dependenciesmissingexception.dependenciesmissingexception(v=VS.90)
 ms:contentKeyID: 22049477
 ms.date: 05/02/2012
+ms.author: mairaw
```
Comment (wadepickett) [inline iis/extensions/database-manager-reference/dependenciesmissingexception-constructor-microsoft-web-management-databasemanager_1.md:9]:
```suggestion
ms.author: johnhart
```

**Comment by wadepickett** [comment]
For ms.author which is internal, it would be johnhart.  for author it would be the github name John-Hart.   Changed those.

**Review by wadepickett — APPROVED** [review]

**Comment by mairaw** [comment]
Given that the author metadata is set globally, why were they also added to a few individual topics?

**Comment by wadepickett** [comment]
I don't know the history. I was just jumping in to unstick a few PR that were sitting there a while waiting for help.  Sourabh would know but he is no longer on IIS.

**Comment by wadepickett** [comment]
I don't know the history. I was just jumping in to unstick a few PR that were sitting there a while waiting for help.  Sourabh would know but he is no longer on IIS.

**Comment by mairaw** [comment]
@damabe added them on this PR, so there's no history.

**Pull request merged**